wording-guidelines.md

```diff
@@ -87,61 +144,121 @@ example on basic syntactic exposition, but do show surprising outcomes
 and interactions with other rules. For syntax disambiguation rules,
 present an example of a tricky parse.
 
+### Normative requirements and recommendations
+
 Prefer normative requirements if possible; this increases portability.
+
 Sometimes, implementations cannot all provide a feature due to
 environmental limitations or other concerns, but the standard should
-still guide common implementations towards a desirable outcome.  For
-that case, prefix a paragraph with "Recommended practice:" and use
-"should".
+still guide common implementations towards a desirable outcome.
+For that case, prefix a paragraph with "*Recommended practice*:" and use "should".
+
+**Example:** ([[intro.abstract] paragraph 6](https://eel.is/c++draft/intro.abstract#6))
 
-When adding a new conversion, also check overload resolution for a
-corresponding adjustment.
+> *Recommended practice*:
+> An implementation should issue a diagnostic when such an operation is executed.
```

Comment:
"executed" and "diagnostic" is a problem for some; I've taken another example.

Reply:
That's okay. Iirc I took that example in particular because it was the first appearance of recommended practice in the draft.